fluentd/lib/fluentd-okd-config-generator/lib/fluentd-okd-config-generator/templates/store_elasticsearch.erb

```diff
@@ -0,0 +1,47 @@
+<store>
+    @type elasticsearch
+    @id <%= store_id(@target, @context) %>
+    host "<%= @target[:host] %>"
+    port "<%= @target[:port] %>"
+    <%- unless @target[:tls_key].nil? -%>
+    scheme https
+    ssl_version TLSv1_2
+    <%- else -%>
+    scheme http
+    <%- end -%>
+    target_index_key viaq_index_name
+    id_key viaq_msg_id
+    remove_keys viaq_index_name
+    user fluentd
+    password changeme
+<% unless @target[:tls_key].nil? %> 
+    client_key "<%= @target[:tls_key] %>"
+    client_cert "<%= @target[:tls_cert] %>"
+    ca_file "<%= @target[:tls_cacert] %>"
```

Comment:
We shouldn't have two separate `unless @target[:tls_key].nil?` sections.
`ca_file` should be in the above

This doesn't matter if client cert auth is the only auth we support, but technically, `client_key` and `client_cert` should be in a separate section about authentication - that is, the user could use `https` with username/password auth or token auth.  But if the user is using client cert auth, they must use `https`.